content/examples/databricks-cluster-local.Rmd

```diff
@@ -0,0 +1,160 @@
+does not apply to your use case, then you can install RStudio Server Pro
+directly within a Databricks cluster.
+
+With this configuration, RStudio Server Pro is installed on the Spark driver
+node and allows users to work locally with Spark.
+
+<img src="/images/deployment/databricks/rstudio-databricks-local.png" width='800px' align='center'/>
+
+This configuration can result in more complex configuration, limited
+connectivity to other storage and compute sources, resource contention between
+RStudio Server Pro and Databricks, and maintenance concerns due to the ephemeral
+nature of Databricks clusters.
+
+For additional details, refer to the
+[FAQ for RStudio in the Databricks Documentation](https://docs.databricks.com/spark/latest/sparkr/rstudio.html#frequently-asked-questions-faq).
+
+## Advantages and limitations
+
+Advantages:
+
+- Can load data from the cluster since RStudio Server Pro is installed within
+  the Databricks cluster
+
+Limitations:
+
+- If the Databricks cluster is restarted or terminated, then the instance of
+  RStudio Server Pro will be terminated and its configuration will be lost
+- If users do not persist their code through version control or the Databricks
+  File System, then you risk losing user's work if the cluster is restarted or
+  terminated
+- RStudio Server Pro (and other RStudio products) installed within a Databricks
+  cluster will be limited to the resources and lifecycle of that particular
+  Spark cluster
+- Non-Spark jobs will use CPU and RAM resources within the Databricks cluster
+- Need to install one instance of RStudio Server Pro per Spark cluster that you
+  want to run jobs on
+
+## Requirements
+
+- Databricks Runtime 4.1 or above
+- The cluster must not have "table access control" or "automatic termination" enabled
+- You must have "Can Attach To" permission for the Databricks cluster
+
+## Preparation
+
+The following steps walk through the process to install RStudio Server Pro
+on the Spark driver node within your Databricks cluster.
+
+The recommended method for installing RStudio Server Pro to the Spark driver
+node is via SSH. However, an alternative method is available if you are not able
+to access the Spark driver node via SSH.
+
+## Configure SSH access to the Spark driver node
+
+Configure SSH access to the Spark driver node in Databricks by following the
+steps in the [SSH access to clusters section of the Databricks Cluster
+configurations
+documentation](https://docs.databricks.com/clusters/configure.html#ssh-access-to-clusters).
+
+**Note:** If you are unable to configure SSH access or connect to the Spark
+driver node via SSH, then you can follow the steps in the [Get started with
+RStudio Server Pro section of the RStudio on Databricks
+documentation](https://docs.databricks.com/spark/latest/sparkr/rstudio.html#get-started-with-rstudio-server-pro)
+to install RStudio Server Pro from a Databricks notebook, then skip to the
+section on "Using sparklyr".
```

Comment:
Section is called "Configuring sparklyr" and it would be nice to link to that section directly from here.

Reply:
Sorry, not following here. This is the section in the Databricks docs for installing and configuring RSP using their "Bash in Python in Databricks notebook" approach. I don't see a sparklyr section on this page?

Were you referring to a different link in this PR or a different Databricks docs page?

Reply:
Oh got it, you meant in this doc. Sorry, I've been staring at this too long. 😄

Reply:
Fixed in b89c94b.